fabfile.py

```diff
@@ -1820,3 +1955,40 @@ def env_bel_bluenove():
     env.uses_ngnix = True
     env.uses_uwsgi = True
     env.gitbranch = getenv("GITBRANCH", "master")
+
+
+""" The various services used by the stack """
```

Comment:
Should we move this to the very top of the file for ease of readability/and immediate invocation of the method below?